CONTRIBUTING.md

```diff
@@ -5,39 +5,42 @@ Please familiarize yourself with the SPDX License List and its supporting docume
 
-## Request a new license or exception be added to the SPDX License List
+# Request a new license or exception be added to the SPDX License List
 The SPDX Legal Team appreciates proposals for new free and open source licenses or exceptions to be added to the SPDX License List.  To request a license or exception be added to the SPDX License List, please:
 
 1.  Review the [license list overview and inclusion principles](https://spdx.org/spdx-license-list/license-list-overview).
 
 2.  Check the [SPDX License List](https://spdx.org/licenses/), historical license and exceptions [tracking page](https://docs.google.com/spreadsheets/d/11AKxLBoN_VXM32OmDTk2hKeYExKzsnPjAVM7rLstQ8s/edit?pli=1#gid=695212681) and issues labeled [new license/exception request](https://github.com/spdx/license-list-XML/labels/new%20license%2Fexception%20request) to ensure this license or exception has not been previously requested. 
 
-3. Submit your request with all of the following information either via a new Issue or via the SPDX-legal mailing list
-(If you are not on the SPDX-legal mailing list, please join first, see above. If you send your license request without joining the mailing list, this will greatly slow down correspondance in both directions.)
-
-* Issue name or email subject line: "New License/Exception Request: <license/exception name>" 
-* Provide a proposed Full Name for the license or exception.
-* Provide a proposed Short Identifier.
-* Provide a functioning URL reference to the license or exception text, preferably from the author or a community recognized source.
-* Indicate whether the license is OSI-approved or is currently under review. (If the latter, please provide some information as to where in the process.) 
-* Provide a short explanation regarding the need for this license or exception to be included on the SPDX License List, and identify at least one program that uses it.
-* Provide the text of the license. 
-
-### Review Process
-1. The SPDX Legal Team will discuss any submissions for new licenses or exceptions on the bi-weekly calls. If further information or clarification is required, the Team will reach out to the requestor accordingly.
-2. Issues will be labled either "new license/exception: Accepted" or "new license/exception: Not Accepted" as appropriate with an explanation and the Issue closed for the latter case.
-3. Upon acceptance, the new license or exception will be prepared by a member of the SPDX Legal Team in the proper XML format and plain text via a Pull Request and reviewed as appropriate. 
+3. Submit your request via the [SPDX Online Tool, Submit New License] (http://13.57.134.254/app/submit_new_license/) using the following guidance, also see [overview-fields] (https://spdx.org/spdx-license-list/license-list-overview#fields) for more info on some of these fields. Note, you must have a Github id in order to use this tool/process. 
```

Comment:
```suggestion
3. Submit your request via the [SPDX Online Tool, Submit New License](http://13.57.134.254/app/submit_new_license/) using the following guidance, also see [overview-fields](https://spdx.org/spdx-license-list/license-list-overview#fields) for more info on some of these fields. Note, you must have a Github id in order to use this tool/process. 
```

Reply:
Very minor change -- I think removing the spaces between the square brackets and parentheses may be necessary in order to make the Markdown links appear correctly.